### PR TITLE
I18n instructions

### PR DESCRIPTION
**Pull request by cap10morgan**
This adds support for translating instructions to English and Spanish, and it requests all translations from election-authority-works when it imports those.

For now it only sends English translations in election create and update events so that notifications don't break. Once we add i18n to notifications, we should remove the filters on those so they get all translations. Those are [here](https://github.com/democracyworks/election-works/blob/i18n-instructions/src/election_works/handlers.clj#L90-L91) and [here](https://github.com/democracyworks/election-works/blob/i18n-instructions/src/election_works/handlers.clj#L108-L109).

I tested this with i18n election-authority-works in krakenstein and got my vote-tomorrow email intact.

[Pivotal card](https://www.pivotaltracker.com/story/show/114350623)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ app:
     - wildfly
     - datomic
 wildfly:
-  image: quay.io/democracyworks/wildfly:8.2.0.Final
+  image: quay.io/democracyworks/wildfly:9.0.2.Final-debug
```
Comment (tie-rack) [inline docker-compose.yml]: What's up with using the debug version?

File: src/election_works/entities/election.clj [hunk 2]
```diff
@@ -240,6 +242,14 @@
                      merge)}
    [:fields]))
 
+(defn entity->instructions
+  [instructions]
+  (when (seq instructions)
+    (reduce (fn [m i]
+              (assoc-in m [(:instruction/type i) (:i18n/language i)]
+                        (:instruction/text i)))
+            {} instructions)))
```
Comment (tie-rack) [inline src/election_works/entities/election.clj]: The `seq` check is essentially performed by `reduce`. The body could be:

```clj
(reduce (fn [m i]
          (assoc-in m [(:instruction/type i) (:i18n/language i)]
                    (:instruction/text i)))
        nil instructions)
```

File: docker-compose.yml (hunk 1 above)
Comment (cap10morgan) [inline docker-compose.yml]: That's what we use everywhere (including production). It logs at the DEBUG level.

**Comment by cap10morgan** [comment]
@tie-rack Ready for re-review.

**Comment by cap10morgan** [comment]
@tie-rack ping

**Comment by tie-rack** [comment]
You are clear for takeoff.

**Comment by cap10morgan** [comment]
We can merge and deploy this after we have merged and deployed [this election-authority-works PR](https://github.com/democracyworks/election-authority-works/pull/29) and re-imported the election authorities w/ the new i18n format (currently in the "i18n States" tab in the Google Sheet).

It will then import voter reg. methods and voting methods from election-authority-works in both languages when importing new elections. But existing elections won't be updated. I don't _think_ we need to worry about that yet as turbovote-web (for better or worse) isn't currently looking at the elections for these strings. That's something we'll need to reconcile eventually (there's little point in allowing an override of those things in elections if we're just going to ignore the overridden values).

ANYway, I'll put together a deployment plan for all of this, because it's becoming a bit complex.

**Pull request merged**